src/components/NewCommentForm.tsx

```diff
@@ -45,24 +139,30 @@ export const NewCommentForm: React.FC = () => {
             name="email"
             id="comment-author-email"
             placeholder="email@test.com"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': hasEmailError })}
```

Comment:
Consider adding email validation logic to ensure that the email address entered is valid. This can prevent invalid email addresses from being submitted.